packages/sql-openapi/lib/shared.js

```diff
@@ -73,6 +86,33 @@ function rootEntityRoutes (app, entity, whereArgs, orderByArgs, entityLinks, ent
       const key = queryKeys[i]
 
       if (key.startsWith('where.')) {
+        if (key.startsWith('where.or')) {
```

Comment:
Can you move this if before the previous one? I don't think there is a need for nesting here.

Reply:
Done!